qa/tasks/scripts/munge_policy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# munge_policy.sh
+#
+# Adds or removes roles from policy.cfg
+#
+
+set -ex
+
+OPERATION=$1 # add_node_as_role / remove_storage_only_node
+NODE_FQDN="${2}" # target192168000063.teuthology
+ROLE="${3}" # storage
+POLICY_CFG="${4}"
```

Comment:
I think POLICY_CFG should be the first argument, since because the ROLE is not used when deleting a node and ROLE can go last...